http-handler/src/main/java/com/flipkart/poseidon/handlers/http/impl/SinglePoolHttpTaskHandler.java

```diff
@@ -112,7 +111,16 @@ public void init(TaskContext taskContext) throws Exception {
             throw new Exception("Invalid pool name specified");
         }
 
-        // create the pool object       
+        // Calculate the maxConnections from poolSize params.
+        int maxConnections = 10; // default as per TaskHandlerExecutorRepository.DEFAULT_THREAD_POOL_SIZE
+        if(getConcurrentPoolSizeParams() != null && !getConcurrentPoolSizeParams().isEmpty()) {
```

Comment:
There is a commandPoolSizeParams as well. We need to see how these two inter-play.